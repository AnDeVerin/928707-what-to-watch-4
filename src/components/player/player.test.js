import React from 'react';
import renderer from 'react-test-renderer';
import Player from './player.jsx';

const MOVIE = {
  thumbUrl: 'img/fantastic-beasts-the-crimes-of-grindelwald.jpg',
  trailer: 'https://example.com/video.mp4',
};

describe('Player component', () => {
  it('renders correctly', () => {
    const component = renderer
      .create(
        <Player isPlaying={true} poster={MOVIE.thumbUrl} src={MOVIE.trailer} />,
        {
          createNodeMock: () => {
            return {
              play: () => {},
            };
          },
        }
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
