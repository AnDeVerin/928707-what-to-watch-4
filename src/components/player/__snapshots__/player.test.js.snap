// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player component renders correctly 1`] = `
<video
  className="player__video"
  muted={true}
  playsInline={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  src="https://example.com/video.mp4"
/>
`;
